### PR TITLE
Include docs for usage with deno

### LINKED ISSUE #26: Add an example for usage with deno
Add an example for usage with deno. FYI #24

### PR DESCRIPTION
**Pull request by mesaugat**
Bonus: Update ES6 version to use named import

👉 https://github.com/denoland/registry/pull/146

Closes #26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Include the package using `require()` or ES2015/TypeScript `import` statement:
 const KeyCode = require('keycode-js');
 
 // ES2015+
-import KeyCode from 'keycode-js';
+import * as KeyCode from 'keycode-js';
```
Comment (kabirbaidhya) [inline README.md]: @mesaugat Will the old syntax still work? I think it still should or it will be a breaking change.

Comment (mesaugat) [reply README.md]: I am not sure it will work because we don't have a default export. Will need to check.

Comment (mesaugat) [reply README.md]: It should work.
https://github.com/kabirbaidhya/keycode-js/blob/master/dist/keycode.cjs.js#L3

Comment (mesaugat) [reply README.md]: Oops it doesn't work. I tried it in a React App. I am doubtful, did it work before this?

Comment (mesaugat) [reply README.md]: It is a breaking change if we need to support default export. Ok, let's come back to this in another thread.

**Comment by mesaugat** [comment]
@kabirbaidhya This is ready to be merged. I've tested it. 

https://deno.land/x/keycode/README.md 👐

File: README.md (hunk 1 above)
Comment (mesaugat) [reply README.md]: Do we need to have a default export? I think we can release this as `2.0.0`

Comment (kabirbaidhya) [reply README.md]: Yes. It's for 2.0.0

**Comment by kabirbaidhya** [comment]
Nice. It's on the registry now; too fast :). Thanks :100:

**Review by kabirbaidhya — APPROVED** [review]

**Pull request merged**